docs/flutterbloccoreconcepts.md

```diff
@@ -70,6 +70,10 @@ BlocProvider.value(
 then from either `ChildA`, or `ScreenA` we can retrieve `BlocA` with:
 
 ```dart
+// with extensions
+context.bloc<BlocA>(context);
```

Comment:
I'm probably missing something here (please forgive my newbieness), but shouldn't it be context.bloc<BlocA>(); (no need to mention context again)?
Same in a few other places in this MR.

Reply:
oops you're totally right! Good catch 🙏